authz/rbac_translator_test.go

```diff
@@ -205,32 +205,6 @@ func TestTranslatePolicy(t *testing.T) {
 				},
 			},
 		},
-		"empty principal field": {
```

Comment:
If we are deleting this test, would a test with `"principals": ["*", ""]` make sense here?

Reply:
Sure. Added test.